### PR TITLE
Ability to return verbose output messages

### PR DESCRIPTION
**Pull request by andrewwinters5000**

### CODE REVIEW

File: src/HOHQMesh.jl [hunk 1]
```diff
@@ -66,7 +70,11 @@ function generate_mesh(control_file;
     flush(tmpio)
 
     # Run HOHQMesh and store output
-    readchomp(`$(HOHQMesh_jll.HOHQMesh()) -f $tmppath`)
+    if ~verbose
+      readchomp(`$(HOHQMesh_jll.HOHQMesh()) -f $tmppath`)
+    else
+      readchomp(`$(HOHQMesh_jll.HOHQMesh()) -verbose -f $tmppath`)
+    end
```
Comment (ranocha) [inline src/HOHQMesh.jl]:
```suggestion
    if verbose
      readchomp(`$(HOHQMesh_jll.HOHQMesh()) -verbose -f $tmppath`)
    else
      readchomp(`$(HOHQMesh_jll.HOHQMesh()) -f $tmppath`)
    end
```
That's significantly easier to read for me, but YMMV

Comment (andrewwinters5000) [reply src/HOHQMesh.jl]: True I'm not sure why I did it the first way TBH

File: test/runtests.jl [hunk 2]
```diff
@@ -12,4 +12,9 @@ using Test
     @test generate_mesh(control_file) isa String
   end
 
+  @testset "generate_mesh()" begin
```
Comment (sloede) [inline test/runtests.jl]:
```suggestion
  @testset "generate_mesh(; verbose=true)" begin
```

**Review by sloede — CHANGES_REQUESTED** [review]
LGTM! I'd only rename the testset to avoid ambiguities. Thanks for adding this feature, @andrewwinters5000!

File: test/runtests.jl (hunk 2 above)
Comment (andrewwinters5000) [reply test/runtests.jl]: Agreed, this was a copy paste error on my part

**Review by sloede — APPROVED** [review]

**Pull request merged**